client/src/components/uploads/UploadUserImage.js

```diff
@@ -37,8 +46,10 @@ const UploadUserImage = ({ userPayload, setUserPayload }) =>{
 						</section>
 					)}
 				</Dropzone>
-
-				<input className="button" type="submit" value="Add" />
+				<p>
+					Preview: <img src={source} />
+				</p>
+				<input className="button" type="submit" value={buttonValue} />
```

Comment:
this makes the registration workflow a bit confusing having 2 different buttons. generally a user would expect to fill out the whole form, attach/upload the image, and press one button at the end to submit the form with the image included. Having another button to "upload the image" adds too many steps. I like the preview, but let's remove the extra upload button. the image in the registration form should be added to state with the rest of the new user data, then have one submit button. they can still attach a new image if they want and don't like the first one, but let's make the process easier by removing an extra step